2-mandatory/4-stocks.js

```diff
@@ -48,7 +58,14 @@ function getAveragePrices(closingPricesForAllStocks) {
     The price change value should be rounded to 2 decimal places, and should be a number (not a string)
 */
 function getPriceChanges(closingPricesForAllStocks) {
-    // TODO
+  let priceChangeStock = [];
+  for (let stockPriceBrand of closingPricesForAllStocks) {
+    let priceChangeBrand = stockPriceBrand[4] - stockPriceBrand[0];
```

Comment:
You could use stockPriceBrand[stockPriceBrand.length-1] here.

Reply:
Thank you